livre/core/render/GLContext.h

```diff
@@ -33,7 +33,7 @@ namespace livre
 class GLContext
 {
 public:
-    LIVRECORE_API GLContext();
+    LIVRECORE_API GLContext( const GLEWContext* glewContext );
```

Comment:
Then you need to use the Qt equivalent of GLEW (forgot the name). The driver developers enforce this, more or less.